<launch>
	<!-- doc ROS node for FT232H -->
	<arg name="serial_no" default="" doc="Serial number of device to use."/>
	<arg name="ns" default="/" doc="Namespace for the device."/>
	<arg name="test" default="false" doc="Flag to test mode if no hardware is attached."/>


	<node name="ft232h_interface" pkg="ft232h_interface" type="ft232h_interface" ns="$(arg ns)" output="screen" args="_test:=$(arg test) _serial_no:=$(arg serial_no)">
		<rosparam command="load" file="$(find ft232h_interface)/config/ee_pump.yaml"/>
	</node>
</launch>
